TRANSLATION.md

```diff
@@ -1,76 +1,25 @@
 # Node.js Website Translation Policy
 
-Node.js is a global platform and so this site has many translations. The translation of the site into
-languages other than English is handled by the localization working group of the language in question. If you
-would like to contribute to the translation of nodejs.org, please refer to the following process:
+We use Crowdin to manage translations. You can join the Node.js project on [Crowdin](https://crowdin.com/project/nodejs-website).
 
-> Since Apr, 1 2020 translation process moved to [Crowdin](https://crowdin.com/project/nodejs-website)
+## How to translate
 
-## Get started
+1. Request to join the [Crowdin](https://crowdin.com/project/nodejs-website)
+2. Select the language you want to translate
+3. Start translating
 
-1. Open [nodejs-website](https://crowdin.com/project/nodejs-website) Crowdin project
-2. Find your locale and start translation. Find more details in [guide for volunteer translators](https://support.crowdin.com/for-volunteer-translators/)
+## How to add a new language
 
-All translated and approved content will be pushed to this repo automatically. You don't need to create any PRs with translation. Just keep localization process on Crowdin.
+Go on `/i18n/config.json` and add the new language to the `locales` array.
 
-Original source can be found in [/locale/en](https://github.com/nodejs/nodejs.org/tree/main/locale/en). If you find any problem with original source, please create a PR with changes directly to `/locale/en`. Crowdin automatically pull all updates within 24 hours.
+Follow the table below to fill the array:
```

Comment:
Why did you remove the JSON snippet? What I was asking was to both keep the JSON snippet, and add the table as a reference of what each field is 😅

Reply:
Oups 😓